### PR TITLE
update sync point - common issue

### PR DESCRIPTION
**Pull request by sanyud24**
updated the common issue of conflicting ports .

closes odk-x/tool-suite-X# https://github.com/odk-x/tool-suite-X/issues/266

addresses odk-x/tool-suite-X#

#### What new issues will need to be opened because of this PR?
           Will be helpful for beginners to setup

### CODE REVIEW

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 1]
```diff
@@ -99,6 +99,7 @@ Setup instructions:
     .. note::
 
       It is important that the right side of the colon stays as 8080. This is the internal port that the web server is looking for.
+      Any other services running on port:80 need to be stopped as it will prevent the server from running ,for ex: Apache2
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-manual-setup.rst:102]: This is helpful content to add! Just a few minor edits. 

"... running," should have comma next to running not "running ," 
please spell out "for ex:" as "for example, ..."

**Comment by sanyud24** [comment]
Ohkay !!

**Review by elmps2018 — APPROVED** [review]

**Comment by elmps2018** [comment]
@sanyud24 this looks good to me. @wbrunette can you review this for content since you commented on the original issue?

**Review by wbrunette — APPROVED** [review]

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 2]
```diff
@@ -99,6 +99,7 @@ Setup instructions:
     .. note::
 
       It is important that the right side of the colon stays as 8080. This is the internal port that the web server is looking for.
```
Comment (wbrunette) [inline odkx-src/sync-endpoint-manual-setup.rst:101]: @sanyud24 excellent work. One thing to point out is that people might need to change from 8080 if another process on their computer is using it. 

So the message would be improved by saying "as 8080 or whatever port you are using (8080 is the default)."

I would also add a reminder at the end to say only "Reminder that only one process can own a port at a time so if another process on the computer is using port 8080 there will be a conflict and sync-endpoint may not function correctly."

**Review by wbrunette — CHANGES_REQUESTED** [review]

**Comment by sanyud24** [comment]
Thankyou , I'll do it now

**Comment by sanyud24** [comment]
Do I have to add the reminder too.
It'll go as:
"Any other services running on port:80 need to be stopped as 8080 or whatever port you are using (8080 is the default), will prevent the server from running ,for ex: Apache2.
"Reminder that only one process can own a port at a time so if another process on the computer is using port 8080 there will be a conflict and sync-endpoint may not function correctly."

**Comment by sanyud24** [comment]
yeah , got it , I'll correct it

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 3]
```diff
@@ -98,7 +98,9 @@ Setup instructions:
 
     .. note::
 
-      It is important that the right side of the colon stays as 8080. This is the internal port that the web server is looking for.
+      It is important that the right side of the colon stays as 8080 or whatever port you are using (8080 is the default). This is the internal port that the web server is looking for.
```
Comment (wbrunette) [inline odkx-src/sync-endpoint-manual-setup.rst:101]: drop the word "internal" leave it simply as "This is the port that"

**Review by wbrunette — CHANGES_REQUESTED** [review]
Sorry I missed this one word change last time. I think it's good after this :) I am ready to approve. I think you want to avoid getting into internal and external ports because it depends on how they have their machine configured. Avoiding using that terminology will simply the message without adding clarification details.

**Comment by sanyud24** [comment]
No , Its alright , I'll fix it

**Review by wbrunette — APPROVED** [review]

**Pull request merged**